en/docs/administer/key-managers/configure-pingfederate-connector.md

```diff
@@ -0,0 +1,285 @@
+            <td>Optional</td>
+          </tr>
+          <tr class="odd">
+            <td><b>Advanced Configurations</b></td>
+            <td>Token Generation</td>
+            <td></td>
+          </tr>
+          <tr class="even">
+            <td>Token Generation</td>
+            <td>Enables token generation via the authorization server.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="odd">
+            <td>Out Of Band Provisioning</td>
+            <td>This enables the provisioning of Auth clients that have been created without the use of the Developer Portal, such as previously created Auth clients.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="even">
+            <td>Oauth App Creation</td>
+            <td>This enables the creation of Auth clients.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="odd">
+            <td><b>Token Validation Method</b></td>
+            <td>The method used to validate the JWT signature.</td>
+            <td></td>
+          </tr>
+          <tr class="even">
+            <td>Self Validate JWT</td>
+            <td>The kid value is used to validate the JWT token signature. If the kid value is not present, `gateway_certificate_alias` is used.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="odd">
+            <td>Use introspect</td>
+            <td>The JWKS endpoint is used to validate the JWT token signature.
+            If this option is used to validate the tokens it is mandatory to add a Token Handling Option.
+                For the Okta it should be <b>JWT</b> and it is required to specify a claim mapping as a unique identifier.
+            </br>e.g.,; Claim Key : iss
+            </br>Claim Value : https://localhost:9031 </td>
+            <td>Optional</td>
+          </tr>
+          <tr class="even">
+            <td><b>Token Handling Options</b></td>
+            <td>Provides a way to validate the token for this particular authorization server.
+            This is mandatory if the Token Validation Method is <b>introspect</b></td>
+            <td></td>
+          </tr>
+          <tr class="odd">
+            <td>REFERENCE</td>
+            <td>The tokens matching a specific regular expression (regEx) is validated.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="even">
+            <td>JWT</td>
+            <td>The tokens matching a specific JWT is validated.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="odd">
+            <td>CUSTOM</td>
+            <td>The token matching a custom pattern is validated.</td>
+            <td>Optional</td>
+          </tr>
+          <tr class="even">
+            <td><b>Claim Mappings</b></td>
+            <td>Local and remote claim mapping.</td>
+            <td>Optional</td>
+          </tr>
+          </table>
```

Comment:
How come we do not have a section on "generate keys using the PingFederate key manager" similar to what we have for okta [1], Auth0, keycloak etc.?

[1] https://apim.docs.wso2.com/en/latest/administer/key-managers/configure-okta-connector/#step-3-generate-keys-using-the-okta-key-manager

Reply:
fixed